fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -2530,7 +2531,7 @@ interval
     ;
 
 unitIdentifier
-    : YEAR | MONTH | WEEK | DAY | HOUR | MINUTE | SECOND | QUARTER
+    : YEAR | MONTH | WEEK | DAY | HOUR | MINUTE | SECOND | QUARTER | MILLISECOND | MICROSECOND
```

Comment:
please also add `MILLISECOND` and `MILLISECOND` to the nonReserved words.

Reply:
thank you and I have added them to the nonReserved words

Reply:
> please also add `MILLISECOND` and `MILLISECOND` to the nonReserved words.
thank you and I have added them to the nonReserved words